src/main/scala/nl.knaw.dans.dd.prepub/DansDataVaultMetadataBlockMapper.scala

```diff
@@ -22,7 +22,7 @@ import scalaj.http.Http
 
 import scala.collection.mutable.ListBuffer
 
-class DansDataVaultMetadataBlockMapper {
+class DansDataVaultMetadataBlockMapper(configuration: Configuration) {
```

Comment:
@ChrisMuller80: Try to avoid passing more than necessary down the call stack. Just passing the pid url is enough here. I will change this after merging this, because for the rest it looks OK and I don't want to keep this issue in review for too long on behalf of a couple of lines of code.